### PR TITLE
Consolidated FAQ content

### PR DESCRIPTION
**Pull request by hauten**
Resolves issue #360 

- Adds new FAQ sub-page under About (spell-checked!)
- Includes new info about removing a repo from the org
- Moves some Contrib & Using GH content to About
- Removes the Contrib and Using GH sub-pages
- Updates About menu accordingly
- (Retains Licensing sub-page/menu)

### CODE REVIEW

File: about/faq.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+4. [Enable two-factor authentication (2FA)](https://github.com/settings/security).
+
+### What is two-factor authentication (2FA)?
+
+*Membership in the @LLNL GitHub organization will soon require that 2FA has been enabled on your GitHub account.* There are several options for configuring 2FA for your GitHub account:
+
+* [YubiKey](https://yubico.com) hardware security keys
+    * Note that the YubiKey deployment (called "MyPass" at LLNL) is in its pilot phase, and not all users have them yet.
+        * YubiKeys are the preferred 2FA option, due to the security of the YubiKey hardware tokens. If you have been issued your LLNL YubiKeys they are highly recommended for securing your GitHub (and other) accounts which support them.
+    * [Learn more about setting up your Yubikey with GitHub.com](https://help.github.com/articles/configuring-two-factor-authentication/#configuring-two-factor-authentication-using-fido-u2f)
```
Comment (LRWeber) [inline about/faq.md:42]: I would suggest changing "Yubikey" to "YubiKey" on this line to match the other instances of the word.

Functionally and visually, things look good to me. I'll leave additional content analysis to @IanLee1521

Comment (hauten) [reply about/faq.md:42]: Fixing

File: about/faq.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+3. Head over to the [@LLNL People](https://github.com/orgs/LLNL/people) page and make your membership `Public`.
+
+### How do I get my repo reviewed and released for GitHub?
+
+All information produced by LLNL must follow the guidance set forth by the LLNL Information Management (IM) Policy for both [initial release {% raw %}<i class="fa fa-lock"></i>{% endraw %}](https://dev.llnl.gov/opensource/releasing/) and [incremental contributions {% raw %}<i class="fa fa-lock"></i>{% endraw %}](https://dev.llnl.gov/opensource/contributing/).
+
+### What is/isn’t allowed to be included in my repo?
+
+Before content is placed into an LLNL GitHub.com repository, it should be reviewed and released via the IM process. Once released, an appropriate open source license and `LLNL-CODE-` release number should be provided and included in the repository.
```
[the remark this answers is not in the document]
Comment (hauten) [reply about/faq.md:80]: Revising to:
How do I get my repo reviewed and released for GitHub? - it must be IMed, links to policies
What is/isn’t allowed to be included in my repo? - no classified/etc bullets
My project is approved for release. Now what? - release number, license, readme, notice

File: about/faq.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+* NO Official Use Only (OUO)
+* NO Health Insurance Portability and Accountability Act (HIPAA)
+* NO Personally Identifiable Information (PII)
+* NO NDA or vendor-proprietary information
+* NO Unclassified Controlled Information (UCI)
+* NO Unclassified Controlled Nuclear Information (UCNI)
+
+When in doubt, contact a Derivative Classifier (DC) and/or IM for further guidance.
+
+In addition, your repo must include README, Notice, and License files. See the [LLNL Software Licensing](licenses) page for details and examples.
```
[the remark this answers is not in the document]
Comment (hauten) [reply about/faq.md:95]: See above revised questions

File: about/faq.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+
+    * If your repo is part of the [RADIUSS project](https://software.llnl.gov/radiuss/), be sure to add the `radiuss` topic along with the appropriate [category tags](https://github.com/LLNL/llnl.github.io/blob/master/radiuss/README.md).
+
+2. Let [Twitter](https://twitter.com/LLNL_OpenSource) followers know your project is available on GitHub. Feel free to tag this handle on your own tweet, or submit a request to [open-source@llnl.gov](mailto:open-source@llnl.gov) so we can tweet on your behalf.
+
+3. Publicize any outreach activities or major milestones related to your project. Examples: You have a paper/poster/presentation accepted at a conference; you're hosting a workshop or webinar; your project is nominated for an award; or you're speaking on a podcast or guest blogging.
+
+3. Publicize any outreach activities or major milestones related to your project. Examples: You have a paper/poster/presentation accepted at a conference; you're hosting a workshop or webinar; your project is nominated for an award; or you're speaking on a podcast or guest blogging.
```
[the remark this answers is not in the document]
Comment (hauten) [reply about/faq.md:129]: Fixed

File: about/faq.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+### How do I contribute news or other content to this website?
+
+Submit a pull request! This website is a GitHub repo just like any other LLNL open source project. News is housed in the [`_posts` directory](https://github.com/LLNL/llnl.github.io/tree/master/_posts), and templates are found in the [LLNL/.github repo](https://github.com/LLNL/.github). See the guidelines below about contributing.
+
+Before contributing, please contact [open-source@llnl.gov](mailto:open-source@llnl.gov) with your idea or if you have questions about whether your proposed content requires the LLNL review and release process.
+
+### How do I remove my repo from the LLNL organization and this website?
+
+A project may no longer actively be developed, or it may outgrow a single repo and require its own organization. A repo’s status (e.g., Public Private, Archived) is managed in GitHub (`settings#danger-zone`) via Settings > Manage Access > Who has access / manage > Danger Zone.
```
[the remark this answers is not in the document]
Comment (hauten) [reply about/faq.md:141]: Revising questions:
What should I do if my repo is no longer actively developed/maintained?
My repo has grown. How do I move it out of the LLNL organization?

**Comment by hauten** [comment]
@IanLee1521 when you have a moment, can you please review my new commit

**Comment by IanLee1521** [comment]
Made a couple of minor changes (https://github.com/LLNL/llnl.github.io/pull/362/commits/9911b04b7c3bd3427c1c6f3eea13dd425e31a0a3). Happy with this now and we'll merge it, thanks for doing this @hauten !

**Pull request merged**